guides/source/testing.md

```diff
@@ -984,8 +1044,8 @@ Sign in helper can be a good example:
 #test/test_helper.rb
 
 module SignInHelper
-  def sign_in(user)
-    session[:user_id] = user.id
+  def sign_in_as(user)
+    post sign_in_url(email: user.email, password: user.password)
```

Comment:
We should try to avoid this, since we end up doing two requests in tests, which might(?) be confusing, specially for new users.
The previous example was simple enough to show the purpose of avoiding duplication.